modules/common/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/common/SqsMessage.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2020 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and
+ * limitations there under.
+ */
+package com.snowplowanalytics.snowplow.storage.spark
+
+import io.circe._
+import io.circe.syntax._
+import io.circe.parser
+import io.circe.generic.semiauto._
+
+import cats.implicits._
+
+import com.snowplowanalytics.iglu.core.{SchemaKey, SchemaVer, SelfDescribingData}
+import com.snowplowanalytics.iglu.core.circe.implicits._
+
+/** A SqsMessage is sent by shredder and read by loader. It contains all the shredded types with their path and type (TSV, JSON) */
+final case class SqsMessage(
```

Comment:
`Sqs` part here is overly-specific. If tomorrow we port GCP Snowflake on same architecture we'll have to change the name. I think it's something like `LoadCommand`.

Reply:
Ok then maybe just `ShreddedInfo` ? I think that the name for a data structure should not be related to the action of the component producing/reading it. I need to update the schema in Iglu Central as well.

Reply:
`ShreddedInfo` is better, I think.

My thinking re `LoadCommand` was inline with implementing poor man's CQRS pattern, where Loader is just a microservice (which it is!) receiving various commands from other microservices, not necessarily Shredder, but potentially Insights UI or users themselves. `Load` is just one of few potential ones, such as `Migrate`, `Unload`, maybe even `Vacuum`.